core/frontend/src/tile/GltfReader.ts

```diff
@@ -1795,6 +1865,34 @@ export abstract class GltfReader {
   }
 
   protected async resolveResources(): Promise<void> {
+    // Load any external images and buffers.
+    await this._resolveResources();
+
+    // If any meshes are draco-compressed, dynamically load the decoder module and then decode the meshes.
+    const dracoMeshes: DracoMeshCompression[] = [];
+
+    for (const node of this.traverseScene()) {
+      for (const meshId of getNodeMeshIds(node)) {
+        const mesh = this._meshes[meshId];
+        if (mesh?.primitives)
+          for (const primitive of mesh.primitives)
+            if (primitive.extensions?.KHR_draco_mesh_compression)
+              dracoMeshes.push(primitive.extensions.KHR_draco_mesh_compression);
+      }
+    }
+
+    if (dracoMeshes.length === 0)
+      return;
+
+    try {
+      const dracoLoader = (await import("@loaders.gl/draco")).DracoLoader;
```

Comment:
Unless the app is loading reality models or glTF from local disk, it wouldn't matter because they wouldn't be able to obtain the content to be decoded in the first place.
The loader package allows the caller to specify alternate location from which to obtain the wasm/js. We can consider a subsequent PR to allow the app to configure that. Then an offline app could bundle them with the rest of its assets.

Reply:
To answer "what happens": an exception is produced and caught, and the graphics are not produced.

Reply:
Sounds good, thanks for the info.  That will be more of a concern for Basis, I think.  Good to hear there is an offline / alternate location that can be specified.